my_sprites.py

```diff
@@ -630,8 +638,70 @@ class Weapon(arcade.Sprite):
             "strength": 10,
             "speed": 1.2,
             "max_usage": math.inf
+        },
```

Comment:
Let's sort this list of weapon data alphabetically so AXE (I think?) is the first one. This makes the list easier to read later on.